### ISSUE TITLE
GUI wallet / QtWebEngine aborts after initial splash messages on Ubuntu 18.04

### ISSUE THREAD

**Issue by Sviluppo718** [aliascash/alias-wallet-archive#276]
I have Alias v4.3.1.0 built on this computer.  The command-line version (aliaswalletd) seems to work but when I try to run aliaswallet it shows the initial splash image which displays while it loads the block index, calculates the chain trust, evaluates the ATXOs, prompts for my passphrase (which I enter) and then crashes ingloriously. 

The following is output to the invoking console:
```
WebEngineContext used before QtWebEngine::initialize() or OpenGL context creation failed.
populatePage
Failed to create OpenGL context for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize 24, redBufferSize -1, greenBufferSize -1, blueBufferSize -1, alphaBufferSize -1, stencilBufferSize 8, samples -1, swapBehavior QSurfaceFormat::SwapBehavior(DoubleBuffer), swapInterval 1, profile  QSurfaceFormat::OpenGLContextProfile(NoProfile)) 
Aborted
```

After entering```$ echo $?``` to see what the return value of the last command run (aliaswallet) was: it outputs 134 .

I've done some searches online to find a solution but I think the problem is I haven't exactly delved into C/C++ programming with Qt with any diligence.

I seem to have all the dependencies noted here: https://github.com/aliascash/documentation/wiki/Build-Ubuntu .  (E.g. ```$ dpkg -s qtwebengine5-dev``` shows the version installed as 5.9.5+dfsg-0ubuntu2 .)

I also had this problem for a few days on another Linux system (running Pop!_OS 20.04 -- which is a fork of Ubuntu) but it seemed to be caused by an update one day that later fixed itself with a later update.

**Comment by HLXEasy** [comment]
The linked documentation needs to be updated. Please try to run the wallet with these dependencies:

```
apt-get update
apt-get install -y \
	libqt5quickwidgets5 \
	qml-module-qtquick2 \
	qml-module-qtwebchannel \
	qml-module-qtwebengine \
	qml-module-qtwebsockets \
	qml-module-qtwebview
```

**Comment by HLXEasy** [comment]
I've pushed some updates to the wiki. Please have a look at https://github.com/aliascash/documentation/wiki/Install-Linux-Daemon and https://github.com/aliascash/documentation/wiki/Install-Linux-UI

**Comment by Sviluppo718** [comment]
I may have been too hasty to open the issue.  The computer was powered down for a couple days, I rebooted today and did the usual system updates and aliaswallet did open without any fuss or Qt problems.  That may have been the problem I encountered before on the other system mentioned (perhaps it needed a reboot after some update in order to to be compiled into the active kernel).

Subsequent to the GUI-wallet working I did find that this system didn't have qtwebchannel and qtwebsockets installed, so that will surely help.  Thanks.  I'm closing this issue.
